tests/installation/upgrade_select.pm

```diff
@@ -7,33 +7,9 @@ sub run() {
 
     # hardware detection can take a while
     assert_screen "select-for-update", 100;
-    send_key $cmd{"next"}, 1;
+    send_key $cmd{"next"}, 5;
     assert_screen "remove-repository", 30;
-    send_key $cmd{"next"}, 1;
-    if (check_screen('network-not-configured', 5)) {
```

Comment:
I'm pretty sure all these lines you're removing here are needed by openSUSE

https://openqa.opensuse.org/tests/62377/modules/upgrade_select/steps/3